packages/config/src/__tests__/ConfigParsing-test.ts

```diff
@@ -136,9 +136,9 @@ describe(getConfig, () => {
       expect(exp.foo).toBe('bar+value');
       expect(exp.name).toBe('rewrote+ts-config-test');
       expect(exp._internal).toStrictEqual({
-        dynamicConfigPath: 'ts/app.config.ts',
+        dynamicConfigPath: `ts${path.sep}app.config.ts`,
```

Comment:
I think there's a `path.*` method for parsing this, would look a bit prettier.

Reply:
`slash` is elegant 👍🏻